dev-lang/duktape/duktape-2.6.0.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="Embeddable Javascript engine"
+DESCRIPTION="Duktape is a small and portable ECMAScript E5/E5.1 \
+implementation.  It isintended to be easily embeddable into C programs, with \
+a C API similar in spirit to Lua's."
+HOMEPAGE="https://duktape.org/"
+COPYRIGHT="2013-2019 Duktape authors"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://duktape.org/duktape-${portVersion}.tar.xz"
+CHECKSUM_SHA256="96f4a05a6c84590e53b18c59bb776aaba80a205afbbd92b82be609ba7fe75fa7"
+
+libVersion="206.20600"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	duktape$secondaryArchSuffix = $portVersion
+	lib:libduktape$secondaryArchSuffix = $libVersionCompat
+	lib:libduktaped$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	duktape${secondaryArchSuffix}_devel = $portVersion
+	devel:libduktape$secondaryArchSuffix = $libVersionCompat
+	devel:libduktaped$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	duktape$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	"
+
+defineDebugInfoPackage duktape$secondaryArchSuffix \
+	$libDir/libduktape.so.$libVersion \
+	$libDir/libduktaped.so.$libVersion
+
+PATCH()
+{
+	# Set install path
+	sed -i "s#INSTALL_PREFIX = /usr/local#INSTALL_PREFIX = $prefix#" \
+			Makefile.sharedlibrary || die "failed to set install path"
```

Comment:
Haven't have a look at the source Makefile, but build fails on 32-bit with mixed "prefix/lib"/"libDir": https://build.haiku-os.org/buildmaster/master/x86_gcc2/logviewer.html?buildruns/1158/builds/2040.log